src/app/app.component.ts

```diff
@@ -70,12 +71,19 @@ function fixUpNav(nav: any, baseUrl: string) {
   templateUrl: './app.component.html'
 })
 export class AppComponent implements OnInit {
+  public isReady = false;
+
   constructor(
     private router: Router,
     private windowRef: SkyAppWindowRef,
     private config: SkyAppConfig,
+    private styleLoader: SkyAppStyleLoader,
     @Optional() private searchProvider?: SkyAppSearchResultsProvider
-  ) { }
+  ) {
+    this.styleLoader.loadStyles().then(() => {
```

Comment:
The red banner is a implementation in host only, not in builder.  We've reserved that to indicate when a site is being served locally.

I'm for visually showing something, but perhaps a console.log is sufficient.

Perhaps @Blackbaud-PaulCrowder has an opinion?